src/applications/drydock/blueprint/DrydockWorkingCopyBlueprintImplementation.php

```diff
@@ -569,14 +571,18 @@ private function applyMerge(
         $lease->setAttribute('workingcopy.vcs.error', $error->toDictionary());
         throw $ex;
       }
+      $interface->execx(
+        'git branch -D `git branch | grep -E "%s-.*"`',
+        self::GIT_BRANCH_PREFIX);
+      return;
```

Comment:
I realised we weren't deleting tmp-branches on success in the fallback case where we use the original behaviour.